instance/models/mixins/database.py

```diff
@@ -76,9 +77,13 @@ def _create_database(cursor, database):
     """
     Create MySQL database, if it doesn't already exist.
     """
+    logger = logging.getLogger(__name__)
```

Comment:
I added the logger in that local scope because I noticed the loggers used by Models are instantiated as instance attributes rather than on the module level and wasn't sure if it was going to mess something up.
But it seems like that should be ok. I'll move this to the module level.